plugin/src/main/java/at/helpch/chatchat/command/IgnoreListCommand.java

```diff
@@ -0,0 +1,40 @@
+package at.helpch.chatchat.command;
+
+import at.helpch.chatchat.ChatChatPlugin;
+import at.helpch.chatchat.api.user.ChatUser;
+import dev.triumphteam.cmd.bukkit.annotation.Permission;
+import dev.triumphteam.cmd.core.BaseCommand;
+import dev.triumphteam.cmd.core.annotation.Command;
+import dev.triumphteam.cmd.core.annotation.Default;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+
+import java.util.stream.Collectors;
+
+@Command("ignorelist")
+public class IgnoreListCommand extends BaseCommand {
+    private final ChatChatPlugin plugin;
+    private final static String IGNORELIST_PERMISSION = "chatchat.ignoreList";
+
+    public IgnoreListCommand(final ChatChatPlugin plugin) {
+        this.plugin = plugin;
+    }
+
+    @Permission(IGNORELIST_PERMISSION)
+    @Default
+    public void ignore(ChatUser sender) {
+        if (sender.ignoredUsers().isEmpty()) {
+            sender.sendMessage(plugin.configManager().messages().notIgnoringAnyone());
+            return;
+        }
+
+        String ignoredPlayers = sender.ignoredUsers()
+            .stream()
+            .map(Bukkit::getOfflinePlayer)
+            .map(OfflinePlayer::getName)
+            .collect(Collectors.joining(", "));
+
+        sender.sendMessage(plugin.configManager().messages().ignoredPlayerList()
+            .replaceText(builder -> builder.matchLiteral("<ignoredPlayers>").replacement(ignoredPlayers)));
```

Comment:
Tags should be formed of only these characters: `a-z 0-9 _ -`. No spaces, no uppercase letters. While currently that is not technically a MiniMessage tag, the idea was to make our own "tags" that would have the same conditions so if in the future we want/need to migrate to using MiniMessage tags, it wouldn't require any work from the plugin users.

Should probably change it to `<ignored_players>`.